backend/routes/api.js

```diff
@@ -4,14 +4,16 @@ const router = express.Router();
 
 const listCalendars = require( '../app/controllers/calendar/listCalendars' );
 const getFreeCalendars = require( '../app/controllers/calendar/getFreeCalendars' );
-const fetchCalendar = require( '../app/controllers/calendar/fetchCalendar' );
-const addEvent = require( '../app/controllers/calendar/addEvent' );
-const deleteEvent = require( '../app/controllers/calendar/deleteEvent' );
+const fetchCalendar = require( '../app/controllers/event/fetchCalendar' );
+const addEvent = require( '../app/controllers/event/addEvent' );
+const deleteEvent = require( '../app/controllers/event/deleteEvent' );
+const updateCalendar = require( '../app/controllers/calendar/updateCalendar' );
 
 router.get( '/calendars', listCalendars );
 router.post( '/calendars/free', getFreeCalendars );
 router.get( '/calendar/:calendar', fetchCalendar );
 router.post( '/calendar/delete/:calendar', deleteEvent );
+router.post( '/calendar/update/:calendar', updateCalendar );
```

Comment:
This has to be refactored - `update` route updates a calendar, while `delete` routes deletes an event. These are 2 different resources.